### PR TITLE
Supporting Python 3.10

### LINKED ISSUE #36: Provide support for Python 3.10
### 📝 Description of the feature

At the moment, CI/CD is only checking on Python 3.7 to 3.9. We should also include support for Python 3.10 but this also requires updating ``ansys-dpf-*`` mainly, since it is the most restrictive in terms of dependencies. 

### 💡 Steps for implementing the feature

Activating commented lines in CI/CD file and updating reqs on ``pyproject.toml`` file

### 🔗 Useful links and references

_No response_

### PR DESCRIPTION
**Pull request by RobPasMue**
Closes #36

### CODE REVIEW

**Review by jorgepiloto — APPROVED** [review]

**Comment by RobPasMue** [comment]
We can't merge it just yet. PyDPF-Post is restricting ``vtk`` to be under 9.1.0 and forcing Python version to be between 3.7 and 3.9

They need to solve that issue before merging.

**Comment by RobPasMue** [comment]
@akaszynski I merged #49 to finally test this PR. Seems to work. We therefore verify our build compatibility with Python 3.7 to 3.10. Ready for approval.

**Pull request merged**
